buildSrc/version.properties

```diff
@@ -2,7 +2,7 @@ opensearch        = 2.12.0
 lucene            = 9.8.0
 
 bundled_jdk_vendor = adoptium
-bundled_jdk = 17.0.8+7
+bundled_jdk = 21.0.1+12
```

Comment:
Does this mean the tests don't run against 17 anymore?

Reply:
Correct, we run against bundled JDK version (21 in this case)

Reply:
>  I was having trouble seeing if we validate against other jre (11,17, 21) or platforms (linux, win) - do we have that setup and I missed it?

We do but not for Gradle check